<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainVinculo</class>
 <widget class="QMainWindow" name="MainVinculo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vínculo e desvinculo de Motorista</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="dtVinculo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
<<<<<<< HEAD
      <pointsize>12</pointsize>
=======
      <pointsize>10</pointsize>
>>>>>>> 36660ec5a6bb8db9ab8c35c63d9d0ae047f8c2d8
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Data Vínculo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="matriculaMotorista">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Matrícula Motorista</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nrFrota">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string> Frota</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoSalvar">
    <property name="geometry">
     <rect>
<<<<<<< HEAD
      <x>120</x>
      <y>320</y>
=======
      <x>50</x>
      <y>340</y>
>>>>>>> 36660ec5a6bb8db9ab8c35c63d9d0ae047f8c2d8
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(52, 235, 119);
alternate-background-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Vincular</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoTelaDesvinculo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>340</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(52, 235, 119);
alternate-background-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Desvincular</string>
    </property>
   </widget>
<<<<<<< HEAD
   <widget class="QPushButton" name="botaoTelaDesvinculo">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>141</width>
      <height>41</height>
=======
   <widget class="QLineEdit" name="nm_motorista_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>121</width>
      <height>21</height>
>>>>>>> 36660ec5a6bb8db9ab8c35c63d9d0ae047f8c2d8
     </rect>
    </property>
    <property name="font">
     <font>
<<<<<<< HEAD
      <family>Segoe UI Black</family>
      <pointsize>18</pointsize>
=======
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nome Motora</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nrPlacaFrota">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>PLACA</string>
    </property>
   </widget>
   <widget class="QLabel" name="Fundo_2">
    <property name="geometry">
     <rect>
      <x>-340</x>
      <y>-60</y>
      <width>871</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/testcaminhao (2).jpeg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>20</pointsize>
>>>>>>> 36660ec5a6bb8db9ab8c35c63d9d0ae047f8c2d8
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
<<<<<<< HEAD
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(176,224,230);
alternate-background-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Desvincular</string>
=======
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Vínculo Desvinculo Motorista</string>
>>>>>>> 36660ec5a6bb8db9ab8c35c63d9d0ae047f8c2d8
    </property>
   </widget>
   <zorder>Fundo_2</zorder>
   <zorder>dtVinculo</zorder>
   <zorder>label</zorder>
   <zorder>matriculaMotorista</zorder>
   <zorder>label_3</zorder>
   <zorder>nrFrota</zorder>
   <zorder>label_4</zorder>
   <zorder>botaoSalvar</zorder>
<<<<<<< HEAD
   <zorder>Title</zorder>
   <zorder>botaoTelaDesvinculo</zorder>
=======
   <zorder>botaoTelaDesvinculo</zorder>
   <zorder>nm_motorista_2</zorder>
   <zorder>nrPlacaFrota</zorder>
   <zorder>label_16</zorder>
>>>>>>> 36660ec5a6bb8db9ab8c35c63d9d0ae047f8c2d8
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
